src/libs/actions/Report.ts

```diff
@@ -3600,14 +3606,14 @@ function prepareOnboardingOptimisticData(
             const taskDescription =
                 typeof task.description === 'function'
                     ? task.description({
-                          adminsRoomLink: `${environmentURL}/${ROUTES.REPORT_WITH_ID.getRoute(adminsChatReportID ?? '-1')}`,
-                          workspaceCategoriesLink: `${environmentURL}/${ROUTES.WORKSPACE_CATEGORIES.getRoute(onboardingPolicyID ?? '-1')}`,
-                          workspaceMembersLink: `${environmentURL}/${ROUTES.WORKSPACE_MEMBERS.getRoute(onboardingPolicyID ?? '-1')}`,
-                          workspaceMoreFeaturesLink: `${environmentURL}/${ROUTES.WORKSPACE_MORE_FEATURES.getRoute(onboardingPolicyID ?? '-1')}`,
+                          adminsRoomLink: `${environmentURL}/${ROUTES.REPORT_WITH_ID.getRoute(`${adminsChatReportID}`)}`,
```

Comment:
I guess from user perspective we will have the same result if we just pass `adminsChatReportID`
```suggestion
                          adminsRoomLink: `${environmentURL}/${ROUTES.REPORT_WITH_ID.getRoute(adminsChatReportID)}`,
```
Could you try to navigate the route with `-1` and `undefined` values, will it act the same?

Reply:
@VickyStash You're right but if I didn't make this change, then eslint gets upset with me for passing an undefined value 😅

Reply:
I think it can be resolved by just updating the type, right?

Reply:
UPD: Similar case is discussed in another PR https://github.com/Expensify/App/pull/54534#discussion_r1901660096